tests/integration/test_monte_carlo.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import pybamm
+import pytest
+
+import pybop
+from pybop import (
+    DREAM,
+    DifferentialEvolutionMCMC,
+    DramACMC,
+    HaarioACMC,
+    HaarioBardenetACMC,
+    MetropolisRandomWalkMCMC,
+    PopulationMCMC,
+    RaoBlackwellACMC,
+    SliceDoublingMCMC,
+    SliceStepoutMCMC,
+    # Grad samplers
+    # NUTS,
+    # HamiltonianMCMC,
+    # MonomialGammaHamiltonianMCMC,
+    # RelativisticMCMC,
+    # SliceRankShrinkingMCMC,
+    # EmceeHammerMCMC,
+    # MALAMCMC,
```

Comment:
Why are these commented out? I would suggest to either remove, uncomment or explain why they are left as commented.